client/src/charts/canada/canada.js

```diff
@@ -187,7 +216,9 @@ export class Canada extends React.Component {
             : `${formatter(tick)}+`,
         active: true,
         id: tick,
-        color: get_graph_color(color_scale(tick)),
+        color: color(interpolateBlues(color_scale(tick)))
+          .copy({ opacity: 0.8 })
+          .toString(),
```

Comment:
Same deal as [here](https://github.com/TBS-EACPD/infobase/pull/1425/files#r835270271), maybe define a function for `(scaled_value) => color(interpolateBlues(scaled_value)).copy({ opacity: 0.8 }).toString()` somewhere so we only need to comment on the special `0.8` value once (or not at all at that point)?

Expected values are between 0 and 1, right? Too bad we'd need [this](https://github.com/microsoft/TypeScript/issues/43505) to properly type that.